src/Tribe/Admin/Columns/Tickets.php

```diff
@@ -138,7 +138,7 @@ protected function get_percentage_string( $post_id, $deprecated = null ) {
 		}
 		// In all other cases, calculate the actual percentage
 		else {
-			$percentage = round( $totals['sold'] * 100 / $stock, 0 );
+			$percentage = round( ( 100 / $totals['capacity'] ) * $totals['sold'], 0 );
```

Comment:
What if capacity is 0? Is that case handled before?

Reply:
This will be handled by the other conditionals above.